mlflow/utils/cli.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+
+import click
+
+import mlflow
+import mlflow.utils.environment
+
+
+@click.group("utils")
+def commands():
+    """MLflow utilities."""
+    pass
+
+
+@click.option("--force", default=False,
+              help="Remove the matching environments without asking if this option is set.")
+@click.option("--env-file", "-e", metavar="PATH", default=None,
+              help="Remove cached conda environment only for the given conda yaml file if set.")
+@commands.command("remove-cached-environments")
+def remove_cached_environments(project_path, force):
+    """
+    Remove cached Conda environments.
+
+    When MLflow executes a project with associated conda environment file it will create a new conda
+    environment defined by this file. This environment is cached based on the content of the conda
+    environment yaml file. This means it is only recreated if the project dependencies change.
+
+    There are two reasons why you might need to remove these files:
+
+    1. To save disk space. The environments are never removed and can consume significant amount of
+       disk space.
+    2. To force recreation of the environment in cases when the caching mechanism failed. This
```

Comment:
Tried to dig into the caching problem a bit (created https://github.com/mlflow/mlflow/issues/525 to track it) - I think having this utility is useful regardless to free up disk space (as the docstring mentions).

Reply:
Thanks!